test/customer_test.rb

```diff
@@ -33,10 +34,11 @@
 end
 
 # TODO: remove the 'x' in front of this block when you start wave 2
-xdescribe "Customer Wave 2" do
+describe "Customer Wave 2" do
   describe "Customer.all" do
     it "Returns an array of all customers" do
       customers = Customer.all
+      p customers
```

Comment:
In the future you should remove debugging prints.  (Though I can tell you were pressed for time this time and it's definitely a low priority thing.)